### ISSUE TITLE
How to get a token?

### ISSUE THREAD

**Issue by hussainweb** [4xposed/SlackInvite.me#1]
First of all, thank you for creating this app. :)

I am trying to add my team and it says I need to get a token from the bottom of https://api.slack.com/web. However, there is no token there. There is a link to register my application and a test token generator. I tried going through registering my application but then it asks for a redirect URL and it is not clear what I should use there.

Maybe things have changed since then. It would also be good to update the page with some more instructions.

**Comment by davidneedham** [comment]
I suspect something changed on Slack's side of things. I've used this several times before without any problem, but I agree with hussainweb that the tokens aren't there anymore.

**Comment by davidneedham** [comment]
I haven't confirmed that it works yet, but I was able to get a token by registering a new application. I filled in a name and my own website for the redirect uri and it let the registration go through. I then grabbed the client id# and used it for the token. So far, it looks like this has worked.

**Comment by hussainweb** [comment]
I tried that but I didn't get any invites yet. I tried using both client id and secret for a token.

**Comment by 4xposed** [comment]
yeah, the Slack API changed a bit since I released Slackinvite.me

I'll try to get the project up to date in the next couple of weeks, sorry for the inconvenience.

**Comment by hussainweb** [comment]
No problem. Thank you for building this.

**Comment by davidneedham** [comment]
Thank you 4xposed!
